modules/metrics/src/main/java/com/spotify/apollo/metrics/semantic/LatencySloTracker.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * -\-\-
+ * Spotify Apollo Metrics Module
+ * --
+ * Copyright (C) 2013 - 2016 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+package com.spotify.apollo.metrics.semantic;
+
+import static com.spotify.apollo.metrics.semantic.What.ENDPOINT_REQUEST_LATENCY_SLO_RATE;
+
+import com.codahale.metrics.Meter;
+import com.spotify.metrics.core.MetricId;
+import com.spotify.metrics.core.SemanticMetricRegistry;
+
+public class LatencySloTracker {
+
+  private final Meter latencyMeter;
+  private final int threshold;
+
+  public LatencySloTracker(MetricId id, SemanticMetricRegistry
+      metricRegistry, Integer threshold) {
```

Comment:
`threshold` should be an `int` if you are going to unbox it to store it in `private final int threshold`